lib/hanami/providers/db/sql_adapter.rb

```diff
@@ -73,7 +73,7 @@ def configure_for_database(database_url)
           )
 
           # Extensions for specific databases
-          if database_url.to_s.start_with?(%r{postgres(ql)*://}) # FIXME: what is the canonical postgres URL?
+          if database_url.to_s.start_with?(%r{postgres(ql)*://})
```

Comment:
Note that while the [PostgreSQL documentation](https://www.postgresql.org/docs/current/app-psql.html#id-1.9.4.21.8) uses `postgresql://`, [Heroku uses `postgres://`](https://devcenter.heroku.com/articles/connecting-heroku-postgres#external-connections-ingress) and in my testing `psql` supports both.

A codesearch through GitHub reveals that both are quite commonplace, so I recommend that we continue to support both ways. Annoying, but it is what it is.